packages/components/src/internal/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -50,16 +52,27 @@ interface AssignmentOptionsInputProps {
     relatedFolders?: List<IssuesRelatedFolder>;
 }
 
+interface RestrictedOptionsProps {
+    coreGroups?: List<Principal>;
```

Comment:
I see this same variable in the state definition below. I assume it doesn't belong here? Is there a coreGroups prop actually passed in here to the component?

Reply:
no it's get's calculated in `componentDidMount`. I was following the same pattern that I saw in `AssignmentOptions`, is this not needed?

Reply:
Ah, I see. Yeah it looks like AssignmentOptions is doing that same thing where it is sharing the props definition so it adds `coreGroups` to the props and state so that it can pass down the state coreGroups as a prop to the child input. In both cases, I think that the loading of the `coreGroups` could be moved to be a state variable in the child component and prevent the need to pass them as props down the tree.